sos/plugins/maas.py

```diff
@@ -56,6 +66,10 @@ def setup(self):
             "apt-cache policy python-django-*",
         ])
 
+        for service in self.services:
+            self.add_journal(units=service)
+            self.add_service_status(service)
```

Comment:
Since both `add_journal` and `add_service_status` methods support lists as the `service` argument, you can simply call:

    self.add_journal(units=self.services)
    self.add_service_status(self.services)

Until you need a dedicated journal logfile for each individual service - then just `add_service_status` can be called to the list.

Reply:
done, thanks for the feedbacks.